packages/cohort_banking/src/app.rs

```diff
@@ -29,16 +29,7 @@ pub struct BankingApp {
 }
 
 impl BankingApp {
-    pub async fn new(config: Config) -> Result<Self, String> {
-        let db_config = DatabaseConfig {
-            pool_size: config.db_pool_size,
```

Comment:
Removed DB related configs from cohort_sdk. We do not need it there because DB connections are concerns for app which use sdk.